### ISSUE TITLE
PolyPen: Unhandled Exception Caught: Exception caught

### ISSUE THREAD

**Issue by KevinPoka** [CGCookie/retopoflow#1268]
**PLEASE READ THE FOLLOWING AND USE THE TEMPLATE TO POST YOUR ISSUE**

*Questions not directly related to the RetopoFlow project are considered general and should be posted elsewhere (e.g. git, blender, python, etc).*

**Before raising the issue, please check the following first:**

- [ ] Read through the README.md file in the branch used to assure the process taken is correct
- [ ] Check the existing [issues](https://github.com/CGCookie/retopoflow/issues) and [pull requests](https://github.com/CGCookie/retopoflow/pulls) to make sure the issue has not already been reported and/or fixed.
- [ ] You can replicate the bug and will provide as much info as possible with blend files, screenshots, logfiles, as well as show the expected result and actual result.

Please prefix your issue name with one of the following: **[BUG]** **[PROPOSAL]** **[QUESTION]**

Issue Template
-----------------------------------

**Issue:**

{*enter your issue here*}

{*add any screenshots of the issue here*}

**How to Reproduce:**

- {*add your list replication steps*}

{*attach any files needed to replicate the issue*}

Automatically Generated Info
-----------------------------------

```
Environment:

- RetopoFlow: 3.4.1
- Build: Self built
- Blender: 3.06.1 blender-v3.6-release 2023-07-17
- Platform: Windows, 10, 10.0.19045, AMD64, AMD64 Family 23 Model 113 Stepping 0, AuthenticAMD
- GPU: {'backend': 'OPENGL', 'device': 'NVIDIA', 'vendor': 'NVIDIA Corporation', 'renderer': 'NVIDIA GeForce GT 1030/PCIe/SSE2', 'version': '4.5.0 NVIDIA 536.23', 'compute_shader_support_get': True, 'max_batch_indices_get': 1048576, 'max_batch_vertices_get': 1048576, 'max_texture_layers_get': 2048, 'max_texture_size_get': 32768, 'max_textures_frag_get': 32, 'max_textures_geom_get': 32, 'max_textures_get': 192, 'max_textures_vert_get': 32, 'max_uniforms_frag_get': 4096, 'max_uniforms_vert_get': 4096, 'max_varying_floats_get': 124, 'max_vertex_attribs_get': 16, 'shader_image_load_store_support_get': True, 'shader_storage_buffer_objects_support_get': True}
- Timestamp: 2023-09-23 12:16:32.796419

Runtime:

- Undo: insert, insert, select box, select box
- Error Hash: f2a383d0764bb804020bc66a2431e8ad

Trace:

- EXCEPTION (<class 'ValueError'>): edges.new(...): found the same (BMVert) used multiple times
-     C:\Users\user\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\retopoflow-master\retopoflow\rfmesh\rfmesh.py
- 000 1717:new_edge() bme = self.bme.edges.new(verts)
-     C:\Users\user\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\retopoflow-master\retopoflow\rf\rf_target.py
- 001 0754:new_edge() return self.rftarget.new_edge(verts)
-     C:\Users\user\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\retopoflow-master\retopoflow\rftool_polypen\polypen_insert.py
- 002 0508:_insert() bme = self.rfcontext.new_edge((bmv0, bmv1))
-     C:\Users\user\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\retopoflow-master\retopoflow\rftool.py
- 003 0208:wrapper() ret = fn(*args, **kwargs)
-     C:\Users\user\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\retopoflow-master\retopoflow\rftool_polypen\polypen_insert.py
- 004 0147:insert() return self._insert()
-     .../common\fsm.py
- 005 0169:wrapped() return fn(*args, **kwargs)

```

**Comment by vxlcoder** [comment]
thanks for the report, @KevinPoka.  I was not able to reproduce this bug, but i believe commit 5589c15 should fix this issue.  you can test it by installing RF v3.4.2beta using this temp link: https://www.dropbox.com/scl/fi/7itaxnh7dmn8op7wpsp5w/RetopoFlow_v3.4.2-beta-GitHub.zip?rlkey=ztrd3stsibrjbmjtk3w8g7w41&dl=0

**Comment by jfranmatheu** [comment]
Closing issue due to prolonged inactivity and lack of issuer information (believed to be fixed with 5589c15, a test build was provided but issuer did not confirm if it fixed the reported issue). If the issue persists, please reopen it. Thank you!
